### PR TITLE
Add InputTimeComp and fix some bugs

### PR DESCRIPTION
**Pull request by yang1666204**
## Summary
1. Add a custom form component `InputTimeComp` to support selecting time units.
2. Add a drawer component instead of jumping to other pages.
3. Fix some bugs.

### CODE REVIEW

File: ui/src/pages/Alert/Event/index.tsx [hunk 1]
```diff
@@ -22,13 +22,22 @@ import type { ColumnsType } from 'antd/es/table';
...
 const { Text } = Typography;
 
 export default function Event() {
   const [form] = Form.useForm();
+  const [drawerOpen,setDrawerOpen] = useState<boolean>(false);
+  const [editRuleName, setEditRuleName] = useState<string>();
   const { data: listAlertsRes, run: getListAlerts } = useRequest(
     alert.listAlerts,
   );
+  const editRule = (rule:string)=>{
```
Comment (powerfooI) [inline ui/src/pages/Alert/Event/index.tsx:36]: This file seems not to be formatted.

**Review by powerfooI — CHANGES_REQUESTED** [review]

File: ui/src/pages/Alert/Event/index.tsx (hunk 1 above)
Comment (yang1666204) [reply ui/src/pages/Alert/Event/index.tsx:36]: Maybe I need a tool that automatically formats before submitting.

Comment (powerfooI) [reply ui/src/pages/Alert/Event/index.tsx]: Yes, you'd better add a step in husky hook before real commit.

**Review by powerfooI — APPROVED** [review]
LGTM

**Pull request merged**